Transfer/Sources/CreateTransferPresenter.swift

```diff
@@ -63,6 +63,7 @@ final class CreateTransferPresenter {
     private var sourceToken: String?
 
     var selectedTransferMethod: HyperwalletTransferMethod!
+    var availableTransferMethods = [HyperwalletTransferMethod]()
```

Comment:
No need to add availableTransferMethods in here. Get it again from TransferMetthodRepository